### DISCUSSION TITLE
Pressure sensor & silicon tube

### DISCUSSION THREAD

**Discussion by gerdb** [cardonabits/haxo-hw#32] [Ideas]
2mm inner diameter fits better to the sensor. 
So I used a short 2mm/6mm silicon tube to the sensor and a longer 3mm/6mm to keep the the pressure/airflow as similar as possible to the original design.

**Comment by jcard0na** [comment]
Hi @gerdb,

Yes, that is a better fit, agreed. 

![image](https://github.com/cardonabits/haxo-hw/assets/676181/0457ec49-7a08-49dd-a483-df3afa71f18f)


At the time it was much easier to source 3mm/6mm than 2mm/6mm but that should work better and save you from [this kludge](https://github.com/cardonabits/haxo-hw/blob/main/docs/assembly.md#13-connect-pipes-to-neck).

The outer 6mm is what is required to insert into the 3D printed neck.

Let me know if you encounter any issues with the 2mm/6mm tube.

Cheers,

**Comment by jcard0na** [comment]
I've prototyped 2mm/6mm as well and seems to work perfectly.  Closing this discussion.
Thanks, @gerdb!
